.travis.yml

```diff
@@ -4,13 +4,11 @@
 language: generic
 
 os: osx
-osx_image: beta-xcode6.1
+osx_image: xcode6.4
 
 env:
   matrix:
     
-    - CONDA_PY=27
-    - CONDA_PY=34
```

Comment:
That looks correct now. (conda-forge dropped PY34 and the pre-selector you added dropped PY27.)